rules.d/60-aspm.rules

```diff
@@ -0,0 +1,20 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION!="add", GOTO="aspm_end"
```

Comment:
shouldn't this be ACTION=="remove" as per https://github.com/systemd/systemd/blob/v247/NEWS#L52 ?

Reply:
@bluca updated, thanks!